packages/office-ui-fabric-react/src/components/pickers/Suggestions/Suggestions.tsx

```diff
@@ -307,19 +308,7 @@ export class Suggestions<T> extends BaseComponent<ISuggestionsProps<T>, ISuggest
 
     let { suggestions } = this.props;
 
-    const TypedSuggestionsItem = this.SuggestionsItemOfProperType;
-
-    // TODO:
-    // Move this styled component in a separate file and make it available to the public API.
-    // This should be done after rewriting pickers to use a composition pattern instead of inheritance.
-    const StyledTypedSuggestionsItem = styled<ISuggestionItemProps<T>, ISuggestionsItemStyleProps, ISuggestionsItemStyles>(
-      TypedSuggestionsItem,
-      suggestionsItemStyles,
-      undefined,
-      {
-        scope: 'SuggestionItem'
-      }
-    );
+    const StyledTypedSuggestionsItem: React.ComponentClass<ISuggestionItemProps<T>> = StyledSuggestionsItem as any;
```

Comment:
nit: i know it's only transient but if we're casting through another types, I think [`unknown`](https://github.com/Microsoft/TypeScript/pull/24439) is preferred

Reply:
`unknown` is not suitable for such case. I polished the code to avoid solo any typing.